dask/dataframe/io.py

```diff
@@ -798,9 +798,12 @@ def from_delayed(dfs, meta=None, divisions=None, prefix='from-delayed',
         An iterable of ``dask.delayed.Delayed`` objects, such as come from
         ``dask.delayed`` These comprise the individual partitions of the
         resulting dataframe.
-    $META
```

Comment:
Why was this here?  I'm not familiar with having `$META` in the docstring.  What is this supposed to mean?